backend/src/main/java/de/neuefische/backend/service/APIService.java

```diff
@@ -8,6 +8,10 @@
 import org.springframework.stereotype.Service;
 import org.springframework.web.reactive.function.client.WebClient;
 
+import java.lang.reflect.Array;
+import java.util.ArrayList;
```

Comment:
```suggestion
import java.lang.reflect.Array;
import java.util.ArrayList;
```
same here, benötigst du die? Sehe hier nur List